carmin-server/server/resources/decorators/__init__.py

```diff
@@ -83,14 +96,15 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
-def admin_only(func):
+@get_db_session
```

Comment:
Discussed and agreed to keep it for the moment. In case we feel like needing extra decorators in the future, we will reconsider its importance further and possibly move it to a utils fonction.